addons/info/src/components/PropTable.js

```diff
@@ -22,20 +22,33 @@ export const multiLineText = input => {
       ));
 };
 
+const determineIncludedPropTypes = (propDefinitions, excludedPropTypes) => {
+  if (excludedPropTypes.length === 0) {
+    return propDefinitions;
+  }
+
+  return propDefinitions.filter(
+    propDefinition => !excludedPropTypes.includes(propDefinition.property)
```

Comment:
You need to import a polyfill on top of this file:

```
import 'core-js/fn/array/includes';
```

See https://babeljs.io/docs/plugins/transform-runtime/
> NOTE: Instance methods such as `"foobar".includes("foo")` will not work since that would require modification of existing built-ins

Reply:
Pulled the polyfill into the `PropTable`